src/main/java/com/es/phoneshop/DAO/ProductDao.java

```diff
@@ -10,15 +10,16 @@
 public interface ProductDao {
     Optional<Product> getProductById(Long id);
 
-    List<Product> findProducts(SortField sortField, SortOrder sortOrder, String query);
+    List<Product> findProducts();
+
+    List<Product> sortProducts(SortField sortField, SortOrder sortOrder, String query);
```

Comment:
sortProducts() contains some query in params
method naming or it's purpose are not correct